bigquery/bigquery_authorized_view_tutorial/main.tf

```diff
@@ -0,0 +1,88 @@
+/**
+* Copyright 2025 Google LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+
+# [START bigquery_authorized_view_tutorial]
+/*
+Creates an authorized view.
+*/
+
+
+/*
+Create a dataset to contain the authorized view.
+*/
+resource "google_bigquery_dataset" "default" {
+  dataset_id  = "authdataset"
```

Comment:
Query: is the dataset "authdataset", or "movie_dataset"? You're assigning dataset access to a different dataset later int his sample. 

You can reference `google_bigquery_dataset.default.dataset_id` in other resources to ensure a consistent name. 

It's helpful to clearly show which is the original dataset and which is the authorised view of the data with clear names (unsure what "movies" is in this context. )

Reply:
I re-named the resources for clarity and tested again to make sure it works.